solutions/usaco-810.mdx

```diff
@@ -152,13 +148,51 @@ public class RestStops {
 			}
 		}
 
-		BufferedWriter bw = new BufferedWriter(new FileWriter("reststops.out"));
-		bw.write(ans + "\n");
-		bw.close();
+		PrintWriter pw = new PrintWriter("reststops.out");
+		pw.println(ans);
+		pw.close();
```

Comment:
we should use buffered writer...

Reply:
?? like 90% of the sols here use pw or kattio